<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qg_2dCameraSettingClass</class>
 <widget class="QDialog" name="qg_2dCameraSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>837</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>qg_2dCameraSetting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
   <item>
    <widget class="QtMyGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGraphicsViews" name="lbl_Window" native="true">
        <property name="styleSheet">
         <string notr="true">border: 1px solid transparent;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="6,1">
     <item>
      <widget class="TechGroupBox" name="groupBox_CameraTest2D">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>2D相机测试</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QScrollArea{
border: 1px solid transparent;
}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>280</width>
             <height>516</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_Init">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="title">
               <string>初始化</string>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="5" column="0" colspan="3">
                <widget class="QPushButton" name="pushButton_sureED_Up">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确定此相机为上相机</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButton_FindDevices">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>查找设备</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0" colspan="3">
                <widget class="QPushButton" name="pushButton_sureED_Location">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确定此相机为定位相机</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0" colspan="3">
                <widget class="QPushButton" name="pushButton_sure2D">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确定此相机为2D识别相机</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="pushButton_OpenDevices">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>打开设备</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="pushButton_CloseDevices">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>关闭设备</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="0" colspan="3">
                <widget class="QPushButton" name="pushButton_sureED_Down">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确定此相机为下相机</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="3">
                <widget class="QComboBox" name="comboBox_CameraID">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="8" column="0" colspan="3">
                <widget class="QPushButton" name="pushButton_sideCamera">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>确定此相机为侧相机</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_Grabbing">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="title">
               <string>图像采集</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButton_StartGrabbing">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>开始采集</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_StopGrabbing">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>停止采集</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_SaveImg">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>图片保存</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_Paramer">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="title">
               <string>参数</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_FrameRate">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLineEdit" name="lineEdit_Gain">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_Exposure">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>帧率</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>曝光</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="QPushButton" name="pushButton_ReadParamer">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>读取参数</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>增益</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QPushButton" name="pushButton_WriteParamer">
                 <property name="font">
                  <font>
                   <family>微软雅黑</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>写入参数</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="qg_JogButton" name="widget_JogButton" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_CameraTest3D">
       <property name="title">
        <string>3D相机测试</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_StopGrabbing_3D">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>停止采集</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_StartGrabbing_3D">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>开始采集</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qg_JogButton</class>
   <extends>QWidget</extends>
   <header>qg_jogbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QGraphicsViews</class>
   <extends>QWidget</extends>
   <header>qgraphicsviews.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtMyGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qtmygroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TechGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">techgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
